massgov/mayflower-patternlab
# Dp 8266  print styles for guides -- Do Not Merge

# A New PR has been included with the PR found here: https://github.com/massgov/mayflower/pull/862
# This PR should not be merged


## Description
This PR cleans up and organizes a bunch of print CSS for Mayflower. This is a work in progress but this is a first pass at an overhaul to cover the majority of the more egregious components.

## Related Issue / Ticket

- [JIRA issue](https://jira.state.ma.us/browse/DP-8266)

## Steps to Test


-  Review each *page* in mayflower and ensure that things are readable. This is a first pass, so it's likely that some components will need some more work down the road. Please feel free to add Jira tickets for components that require further work to be perfect. Small tweaks for overlapping or otherwise unreadable text we should fix in this PR. 
- I'd recommend using Chrome Dev Tools to test print styles. 
    - To enable those, open chrome dev tools, open the console drawer (`esc` key) using the menu within that drawer, open the rendering tab, then select print rendering from the bottom dropdown in that list. This is confusing to find... so see attached gif for reference!

![chrome-print-dev-tools](https://user-images.githubusercontent.com/1397914/39708924-7554768e-51e6-11e8-8de6-58dcddc6fb87.gif)


## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*  `_print.scss`

#### @TODO


* Action finders specifically are kind of gross and should have a unique print treatment I think - though I believe they are passable for now. 

* We may want to consider saving ink and avoiding having colored backgrounds wherever possible.
